layouts/page/pulumi-up.html

```diff
@@ -1,39 +1,105 @@
 {{ define "main" }}
-    <div class="pulumi-up-body text-white py-12">
-        <div class="container mx-auto">
-            <section id="about">
+    <div class="pulumi-up-body text-white pb-12 ">
+        <div class="relative overflow-hidden pulumi-up-video-section">
+            <video class="pulumiup-video" autoplay muted loop>
+                <source src="/video/pulumiup-lofi.mp4" type="video/mp4">
+            </video>
+            <div class="pulumiup-title">
                 <div class="neon-container">
                     <span class="blue-neon-sign-xl">Pulumi</span>
                     <span class="green-neon-sign-xl">UP</span>
                 </div>
-                <pulumi-date-countdown
-                    value-label-class="text-2xl"
-                    text-class="date-countdown"
-                    date-string="{{ .Params.countdown_date }}"
-                ></pulumi-date-countdown>
-                <div class="bg-green"></div>
+                <pulumi-mp3-mute-controller
+                    paused-text="FEEL THE VIBE"
+                    playing-text="PAUSE THE VIBE"
+                    audio-element-id="pulumiup-background-music"
+                    file="http://music-bucker-d3204f7.s3-website-us-west-2.amazonaws.com/pulumiup-background-music.mp4"
```

Comment:
I _think_ you'll need this to be `https` in order to avoid a mixed-content issues when this goes out to pulumi.com. 

Also, any reason not to put the video file into the same bucket? Seems like a good use case for a separate ad-hoc media stack of some kind, actually. And where is this managed? pulumi/home? 

Lastly, we talked about trimming this file down, since it's currently sitting at around ~90MB file, and it'd be nice if we didn't have to push that all that data down to mobile devices without users' awareness. Happy to help you edit this if you want.